docs/cosa-significa-essere-un-dev/README.md

```diff
@@ -0,0 +1,97 @@
+
+Oppure si potrebbe dire che "_Essere un dev significa essere persone altamente competenti nel campo informatico; figure prettamente tecniche e verticalizzate in uno o più linguaggi di programmazione_"...ma anche qui, **sarebbe come indossare un maglione troppo stretto**: dopo un po' pizzica.
+
+Potremmo continuare dicendo che "_Essere un dev significa essere un'artigiano del mondo digital: un professionista in grado di trovare la soluzione ad uno specifico problema ed essere in grado di applicarla in un contesto digitale_".
+
+O ancora, volendola vedere sotto un punto di vista più etico: "_Essere un dev significa abbracciare il movimento Open Source, apportando il proprio contributo senza pretendere nulla in cambio e contribuendo così a far crescere la community stessa, oltre che il proprio know-how_".
+
+La verità, caro lettore, è che essere un dev significa essere tante cose contemporaneamente.
+
+Tante _definizioni_, se così vogliamo chiamarle, che però affondano le loro radici su dei principi solidi e genuini: [**passione**](#la-passione-il-combustibile-per-eccellenza), [**curiosità**](#la-curiosità-il-potere-della-scoperta), [**empatia**](#lempatia-la-capacità-di-impersonare-il-prossimo) e [**contaminazione**](#la-contaminazione-fare-squadra-nel-modo-giusto); principi su cui, peraltro, nasce questo libro.
+
+Caratteristiche che nulla hanno a che fare con la scrittura del codice, né presentano alcun tipo di legame diretto con specifiche competenze tecniche; nonostante ciò, risultano essere caratteristiche **fondamentali** per intraprendere questa carriera al meglio, lasciarsi coinvolgere da essa e riuscire a portare valore in questo settore.
+
+## La passione: il combustibile per eccellenza
+
+Decidere di "diventare un dev" significa scegliere di "sacrificare" buona parte del proprio tempo al fine di raggiungere un'obiettivo; non sarà un percorso facile, tantomeno breve; ci saranno momenti di eccitazione, ma anche di sconforto; ci saranno giorni in cui tutto sembrerà andare per il verso giusto, ed altri in cui ogni nostra singola attività verrà minata dalle difficoltà che ci si pareranno davanti; ci saranno concetti che domeremo con estrema facilità, ed altri in cui ne usciremo sconfitti e con la coda tra le gambe.
+
+Fà tutto parte del gioco: _abbraccialo, comprendilo e fallo tuo_; magari non oggi e forse nemmeno domani, ma arriverà il momento in cui riuscirai dove prima hai fallito.
+
+Solitamente, quando ci appassioniamo a qualcosa tendiamo a dedicare molto del nostro tempo a quella specifica attività; la coltiviamo negli anni prendendocene cura, approfondendo ogni aspetto e cercando di scoprire ogni singola caratteristica che possiede...in buona sostanza: _la rendiamo nostra_.
```

Comment:
Trasformerei i tre puntini in un ";" con uno spazio dopo

Reply:
Io sono piu propenso per l'uso del punto piuttosto che del punto e virgola. Cosi la "responsabilita" della frase rimane una sola. E non tre. In oltre il punto ferma la frase li e da un gran peso a quel che si legge.

Reply:
Anche, sì!

Reply:
Dal mio punto di vista, è una questione di scrittura e di stile. Un po' come ogni autore/autrice ha la sua penna preferita, qui si tratta di scegliere anche il modo in cui lo si vuole leggere e soprattutto come si vuole che venga letto. In realtà, sono corrette tutte e tre le forme dal punto di vista sintattico, anche se, riconoscendo il tono "romantico" e informale del capitolo, i tre puntini (senza abusarne), ci stanno!